<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Annotations</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Image</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="_pText">
        <property name="text">
         <string>Text</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="_pPoint">
        <property name="text">
         <string>Point</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="_pLine">
        <property name="text">
         <string>Line</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="_pRectangle">
        <property name="text">
         <string>Rectangle</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="_pEllipse">
        <property name="text">
         <string>Ellipse</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="_pCircle">
        <property name="text">
         <string>Circle</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="_pClearButton">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <widget class="PiiImageDisplay" name="_pResultImageDisplay" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>538</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="_pActionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="_pActionSelectImages">
   <property name="text">
    <string>Select images...</string>
   </property>
  </action>
  <action name="_pActionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionEka">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Eka</string>
   </property>
  </action>
  <action name="actionToka">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Toka</string>
   </property>
  </action>
  <action name="actionKolmas">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Kolmas</string>
   </property>
  </action>
  <action name="actionNelj_s">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Nelj√§s</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PiiImageDisplay</class>
   <extends>QWidget</extends>
   <header location="global">PiiImageDisplay.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>_pActionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
